src/SDKs/OperationalInsights/DataPlane/OperationalInsights/OperationalInsights.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <PropertyGroup>
+    <Description>Microsoft Azure Operational Insights Data Library</Description>
+    <PackageId>Microsoft.Azure.OperationalInsights</PackageId>
+    <Description>Provides Microsoft Azure Operational Insights data operations</Description>
+    <AssemblyName>Microsoft.Azure.OperationalInsights</AssemblyName>
+    <VersionPrefix>1.0.0.0-preview</VersionPrefix>
+    <PackageTags>Microsoft Azure OperationalInsights;OperationalInsights</PackageTags>    
```

Comment:
@csuich2 missing package release notes like below
https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/BatchAI/Management.BatchAI/Microsoft.Azure.Management.BatchAI.csproj#L10